src/dbt_score/cli.py

```diff
@@ -102,14 +118,33 @@ def lint(
         config.overload({"rule_namespaces": namespace})
     if disabled_rule:
         config.overload({"disabled_rules": disabled_rule})
+    if fail_project_under:
+        config.overload({"fail_project_under": fail_project_under})
+    if fail_any_model_under:
+        config.overload({"fail_any_model_under": fail_any_model_under})
 
     if run_dbt_parse:
         dbt_parse()
 
     try:
-        lint_dbt_project(
+        evaluation = lint_dbt_project(
             manifest_path=manifest, config=config, format=format, select=select
         )
+
+        if evaluation.min_model_score < config.fail_any_model_under:
```

Comment:
There might be some edge cases where rounding will lead to nonsensical output, like:

Given:
```
min_model_score = 8.65
fail_any_model_under = 8.69
```

```
Individual model score 8.7 is less than `fail_any_model_under` setting of 8.69
``` 

We can consider using only raw/rounded values (both in calculation and output) or making the rounding strategy a bit more advanced.

I do not think this should block this PR but it's definitely something to keep in mind 🙌

Reply:
That makes sense. I left the rounding the same as the rounding in the actual scoring.